src/clique_table.erl

```diff
@@ -91,7 +91,9 @@ resize_row(MaxLength, Widths) ->
     Sum = lists:sum(Widths),
     case Sum > MaxLength of
 	true ->
-	    resize_items(Sum, MaxLength, Widths);
+	    NewWidths = resize_items(Sum, MaxLength, Widths),
+	    % go around again, just in case some columns were unshrinkable
+	    resize_row(MaxLength, NewWidths);
```

Comment:
Yeah, should probably break if Widths == NewWidths. The column resizing in general seems a little weird.